### PR TITLE
[FIX][business_requirement] Name formating improvement

### PR DESCRIPTION
**Pull request by martintorresvm**
To fix https://github.com/OCA/business-requirement/issues/114

### CODE REVIEW

**Comment by martintorresvm** [comment]
@elicoidal @seb-elico

cc @sudhir-serpentcs

**Review by elicoidal — APPROVED** [review]

**Comment by elicoidal** [comment]
@pedrobaeza @dreispt @seb-elico Let's move forward?

File: business_requirement/models/business.py [hunk 1]
```diff
@@ -251,9 +251,11 @@ def name_get(self):
         result = []
         for br in self:
             if br.ref:
-                formatted_name = '[{}] {}'.format(br.ref, br.description)
```
Comment (pedrobaeza) [inline business_requirement/models/business.py:262]: The solution is to put:
```
formatted_name = u'[{}] {}'.format(br.ref, br.description)
```
not to make the other change

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: business_requirement/models/business.py (hunk 1 above)
Comment (martintorresvm) [reply business_requirement/models/business.py:262]: Sure, thanks for review!

Comment (martintorresvm) [reply business_requirement/models/business.py:262]: done

**Comment by elicoidal** [comment]
@pedrobaeza your comment is attended so I move forward :). Thanks for the review!

**Pull request merged**